internal/web_bundle/create_webpack_config.js

```diff
@@ -68,6 +68,7 @@ module.exports = (
 
   const base = path.join(process.cwd(), sourceDir)
   const gendir = path.join(base, process.env["GENDIR"], '..', 'bin')
+  const gendir2 = path.join(base, process.env["GENDIR"])
```

Comment:
I wonder if we should instead simply update `gendir` by removing `'..', 'bin'`?

If we indeed need both, then it would be worth renaming each to better represent what it's used for, and add a comment with an example where it would be necessary.

Bonus points if you add an example running in CI that highlights the need for this 😅